python/Ganga/test/GPI/FileTests/TestMassStorage.py

```diff
@@ -0,0 +1,304 @@
+from __future__ import absolute_import
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+import tempfile
+import datetime
+import time
+import random
+import os
+import shutil
+import copy
+
+def generateUniqueTempFile( ext = '.txt' ):
+    """ Generate a unique file with a given filename with some random contents and return the name of the file on disk
+    Args:
+        ext (str): This is the extension (including '.') to give to the file of interest
+    """
+
+    myFile = tempfile.NamedTemporaryFile(mode='w',delete=False)
```

Comment:
`NamedTemporaryFile` takes a suffix argument so you don't have to rename the file afterwards